### DISCUSSION TITLE
OR Student looking to contribute

### DISCUSSION THREAD

**Discussion by TeoSkondras** [coin-or/pulp#647] [Q&A]
Hello! I am interested in contributing to the project, is there anything that i could start working on?

**Comment by pchtsp** [comment]
Hello! Great. You can check the contribute guide: https://coin-or.github.io/pulp/develop/contribute.html
And then look for issues with the label "good-first-issue" or "help wanted"

**Reply by TeoSkondras** [reply]
> Hello! Great. You can check the contribute guide: https://coin-or.github.io/pulp/develop/contribute.html And then look for issues with the label "good-first-issue" or "help wanted"

@pchtsp I was thinking about adding a new solver regarding VehicleRoutingProplem type problems, I have experience with those and Python. Should I go for it?

**Reply by TeoSkondras** [reply]
Generally I am interested in implementing a new solver, whether it is about VRP or generally mathematical programming (ex. Hungarian Algorithm)
